libs/langchain-mcp-adapters/__tests__/tools.test.ts

```diff
@@ -130,6 +130,56 @@ describe("Simplified Tool Adapter Tests", () => {
       });
     });
 
+    test("should load tool with no input parameters", async () => {
+      // Set up mock response
+      mockClient.listTools.mockReturnValueOnce(
+        Promise.resolve({
+          tools: [
+            {
+              name: "weather",
+              description: "Get the current weather",
+              inputSchema: {
+                type: "object",
+              },
+            },
+          ],
+        })
+      );
+
+      mockClient.callTool.mockImplementation((params) => {
+        return Promise.resolve({
+          content: [
+            {
+              type: "text",
+              text: `It is currently 70 degrees and cloudy.`,
+            },
+          ],
+        });
+      });
+
+      // Load tools
+      const tools = await loadMcpTools(
+        "mockServer(should load tool with no input parameters)",
+        mockClient as Client
+      );
+
+      // Verify results
+      expect(tools.length).toBe(1);
+      expect(tools[0].name).toBe("weather");
```

Comment:
Given the change that this is meant to test, I'd expect an assertion here that validates that the type of LangChain tool that's returned by `loadMcpTools` is what you'd expect.

Reply:
I'm not sure I see how this can be done, as `loadMcpTools` returns a list of tools with the same interface?
There are no differences between tools with or without parameters besides the schema

Reply:
One is an instance of `DynamicTool`, the other is an instance of `DynamicStructuredTool`.